config/settings.yml

```diff
@@ -15,6 +15,7 @@ feature_flags:
   bibliography_resource: false
   index_related_content: false # A feature that will index related SDR content. Initial implemented for Parker's IIIF AnnotationLists
   add_resource_type_index_field: false # Should the resource type field be added as an index/show field. For some exhibits this is desirable
+  add_parker_search_fields: false # Parker specific search fields
```

Comment:
Just trying to think through the implications of this and wondering a) is this a flag that we will intend to remove at some point, and b) should we use an exhibit specific flag for this?

Reply:
Hmm. I'll try and answer your questions here:

a) Frankly I don't know this answer. I think this will likely set a pattern if there are any other super custom search fields that need to be added this pattern could used. Do you have a suggestion?
b) I thought I was kind of doing it that way... a global false, followed by shared_configs PR's that whitelist parker exhibits (already submitted)

Reply:
I guess I kinda worry about `a` being abused as a pattern (don't have any suggestions otherwise that we can implement w/i our timeframe).

👍 on `b` I hadn't seen that PR.

Reply:
Sure understood. Any suggestions about mitigating `A` ?

Reply:
Sadly no.  😢